### PR TITLE
Update templates to support marking target word with `<strong>`

### PR DESCRIPTION
**Pull request by ghost**
Anki 2.1.50 brought a change to the WYSIWYG note editor.  Previously,
pressing `Control+b` or the `Bold text` button wrapped the selection in
a `<b>` tag, and the latest update changed it to `<strong>`, therefore
breaking scripts and styles for newly added notes.  This patch adds
support for the new tag in affected templates.

-----------

I think that @Feripe-a and @cyphar should also have a vote on this since I'm altering their templates.

I'll fix my own template in another PR, because it needs to be updated anyways.

### CODE REVIEW

**Comment by tatsumoto-ren** [comment]
Anki devs must revert the change.

**Comment by cyphar** [comment]
I wrote a comment on the 2.1.51-rc1 thread, but I don't have an issue with making the templates work with both `<b>` and `<strong>` (you also fixed up some of my JS, which is appreciated :smile:).

File: templates/Japanese fallback/Production/front.html [hunk 1]
```diff
@@ -13,12 +13,9 @@
 
 <script>
     function threeDots() {
-        const sent = document.getElementsByClassName("production")[0];
-        if (sent) {
-            for (const hidden of sent.getElementsByTagName("b")) {
-                hidden.style.visibility = "visible";
-                hidden.innerText = "【" + hidden.innerText.replace(/[^ぁ-んァ-ン]/g, "") + "】";
-            }
+        for (hidden of document.querySelectorAll(".production b, .production strong")) {
```
Comment (tatsumoto-ren) [inline templates/Japanese fallback/Production/front.html]: You also can add `const` here I think to revert the regression.

**Comment by tatsumoto-ren** [comment]
👍

File: templates/Japanese fallback/Production/front.html (hunk 1 above)
Comment (ghost) [reply templates/Japanese fallback/Production/front.html]: Yep, that was a silly omission of mine. I'll squash the commits before merging.

Comment (ghost) [reply templates/Japanese fallback/Production/front.html]: Well, my comment was a little bit late :^)

Comment (tatsumoto-ren) [reply templates/Japanese fallback/Production/front.html]: Probably not a big deal for two commits.

**Pull request merged**